torchvision/transforms/functional.py

```diff
@@ -446,22 +446,47 @@ def _get_perspective_coeffs(startpoints, endpoints):
     return res.squeeze_(1).tolist()
 
 
-def perspective(img, startpoints, endpoints, interpolation=Image.BICUBIC):
+def perspective(img, startpoints, endpoints, interpolation=Image.BICUBIC, fill=None):
     """Perform perspective transform of the given PIL Image.
 
     Args:
         img (PIL Image): Image to be transformed.
         startpoints: List containing [top-left, top-right, bottom-right, bottom-left] of the orignal image
         endpoints: List containing [top-left, top-right, bottom-right, bottom-left] of the transformed image
         interpolation: Default- Image.BICUBIC
+        fill (n-tuple or int or float): Pixel fill value for area outside the rotated
+            image. If int or float, the value is used for all bands respectively.
+            Defaults to 0 for all bands. This option is only available for ``pillow>=5.2.0``.
     Returns:
         PIL Image:  Perspectively transformed Image.
     """
+    def parse_fill(fill, num_bands):
+        if PILLOW_VERSION < "5.2.0":
```

Comment:
Nope. `Image.transform()` has the `fillcolor` option [since `5.0.0`](https://pillow.readthedocs.io/en/stable/releasenotes/5.0.0.html#image-transform).